inbox/react/components/inbox.mdx

```diff
@@ -2,10 +2,119 @@
 title: '<Inbox />'
 ---
 
+By default, the `<Inbox />` renders a bell button, that opens a popover on click. The popover contains the notifications list and the user preferences.
 
-Renders `<Inbox />` with a bell button, that opens a popover on click. The popover contains the notifications list and the user preferences.
+```tsx
+import { Inbox } from '@novu/react';
+
+function Novu() {
+  return (
+    <Inbox
+      applicationIdentifier="YOUR_APPLICATION_IDENTIFIER"
+      subscriberId="YOUR_SUBSCRIBER_ID"
+    />
+  );
+}
+```
+
+## Navigation 
+
+The Inbox component uses the `routerPush` prop to make your notifications navigatable. We will automatically pass the `redirect.url` from your workflow definitions to the `routerPush` prop.
+
+To make the navigation work, you will need to specify the `routerPush` behaviour depending on your routing library.
+
+<CodeGroup>
+    ```tsx Next.js
+    import { Inbox } from '@novu/react';
+    import { useRouter } from 'next/navigation'
+
+    function Novu() {
+        const router = useRouter();
+
+        return (
+            <Inbox
+                applicationIdentifier="YOUR_APPLICATION_IDENTIFIER"
+                subscriberId="YOUR_SUBSCRIBER_ID"
+                routerPush={(path: string) =>  router.push(path)}
+            />
+        );
+    }
+    ```
+
+    ```tsx Remix
+    import { Inbox } from '@novu/react';
+    import { useNavigate } from '@remix-run/react';
+
+    function Novu() {
+        const navigate = useNavigate();
+
+        return (
```

Comment:
Please align the indentation to two spaces

Reply:
Done, also added a settings.json to make sure it's always 2 tabsize for the team